ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/LdaptiveUserDao.java

```diff
@@ -458,7 +456,7 @@ private void modifyGroupMembership(User user, Group group, AttributeModification
             modifyEntry(conn, groupDn, attributeModificationType, userDn, LDAP_ATTR_MEMBER);
 
         } catch (final LdapException e) {
-            throw Throwables.propagate(e);
+            throw convertLdapException(e);
```

Comment:
down the line I want to try and actually classify the exception being thrown, so I centralized the simple conversion logic to make it easier to parse and convert more intelligently in the future.